### DISCUSSION TITLE
How to unsetMark immediately after input is updated

### DISCUSSION THREAD

**Discussion by goergemiguel** [ueberdosis/tiptap#2595] [General Questions] [answered]
Hi guys, heres my current scenario:

initial state is `<p><b>i am bold sentence</b></p>` and when i click on the editor and continue typing, the output will be something like below where the new input is also inside the <b> tag.

`<p><b>i am bold sentence, this is what i am typing after</b></p>`

What i want to achieve is to have 

`<p><b>i am bold sentence</b>, this is what i am typing after</p>`

and i've tried manually calling `editor.commands.unsetBold` on `onUpdate`, `onCreate`, `onFocus` events but seems it's not working as intended or i'm missing something here? 

Any help would be appreciated, thanks in advance!

```
	mounted() {
		this.editor = new Editor({
			extensions: [
				Document,
				Paragraph,
				Heading,
				Text,
				Bold,
			],
			content: "<p><b>i am bold sentence</b></p>",
			onCreate: ({ editor }) => {
				console.log("on creating")
				editor.commands.unsetBold() // not working
			},
			onUpdate: ({ editor }) => {
				editor.commands.unsetBold() // not working
				console.log(editor.getHTML())
				this.$emit("update:modelValue", editor.getHTML())
			},
		})
	},
```

**Comment by cadars** [comment, marked as answer]
This? https://tiptap.dev/api/schema#inclusive

**Reply by rfgamaral** [reply]
Yes, that's what you're looking for.
